config_samples/bsc/mainnet/wsteth_mainnet_config_binance.json

```diff
@@ -0,0 +1,28 @@
+{
+  "contracts": {
+    "0x6981F5621691CBfE3DdD524dE71076b79F0A0278": "ERC1967Proxy",
+    "0xe82c2a5846cfb6d8683d6b636719e7aa61486838": "NttManager",
+    "0xbe3F7e06872E0dF6CD7FF35B7aa4Bb1446DC9986": "ERC1967Proxy",
+    "0xb948a93827d68a82F6513Ad178964Da487fe2BD9": "WormholeTransceiver",
```

Comment:
It clashes with the ETH NTT Manager proxy, but I guess there is nothing we can do...

Reply:
yeah, the thing sucks